pkg/types/relayer.go

```diff
@@ -16,12 +16,20 @@ type PluginArgs struct {
 }
 
 type RelayArgs struct {
-	ExternalJobID uuid.UUID
-	JobID         int32
-	ContractID    string
-	New           bool // Whether this is a first time job add.
-	RelayConfig   []byte
-	ProviderType  string
+	ExternalJobID      uuid.UUID
+	JobID              int32
+	ContractID         string
+	New                bool // Whether this is a first time job add.
+	RelayConfig        []byte
+	ProviderType       string
+	MercuryCredentials MercuryCredentials
```

Comment:
I had been thinking that this would be hidden inside of the opaque `RelayConfig []byte`, rather than be exposed for everyone like this. We might have to reconsider other approaches.
cc @krehermann @cedric-cordenier